internal/graph/graph.go

```diff
@@ -201,89 +180,88 @@ func (g *ConnectedObjectGraph) findIngresses(
 		return nil, err
 	}
 
-	return g.findIngressesWithTargetRewrite(
+	return g.getRelationshipEdgesWithTargetRewrite(
```

Comment:
For consistency, please either use `relationshipEdgesWithTargetRewrite` here (like you did above) or switch what is above to `g.getRelationshipEdges(..)`.